### PR TITLE
Use policy document as given str

### PR DESCRIPTION
**Pull request by oharan2**

### CODE REVIEW

File: clouds/aws/aws_roles/aws_roles.py [hunk 1]
```diff
@@ -22,30 +20,21 @@ def iam_client(region=DEFAULT_AWS_REGION):
...
+    role_name, policy_name, policy_document, region=DEFAULT_AWS_REGION
 ):
     """
     Create a new policy role or update an existing one.
 
     Args:
         role_name (str): role policy name
         policy_name (str): policy name
-        policy_document_path (str): path to Json file that holds the policy documents
+        policy_document (str): loaded json file that holds the policy documents
```
Comment (myakove) [inline clouds/aws/aws_roles/aws_roles.py:31]: `loaded` what do you mean by this?
Maybe  `policy documents as Json content`  ?

**Review by myakove — CHANGES_REQUESTED** [review]

File: clouds/aws/aws_roles/aws_roles.py (hunk 1 above)
Comment (oharan2) [reply clouds/aws/aws_roles/aws_roles.py]: Done

**Review by rnetser — APPROVED** [review]

**Review by myakove — APPROVED** [review]

**Pull request merged**